packages/serverless/src/domain/use-case/transaction.ts

```diff
@@ -46,6 +56,8 @@ export async function enqueueTransaction(
       merkleHeight: settings?.merkleHeight,
     })
   );
+
+  return insertedId;
 }
 
 export async function getTransaction(
```

Comment:
the return type does not match with graphql
in my case: 
from 
```ts
return transaction
```
to 
```ts
return {
          databaseName: transaction.databaseName,
          transactionType: transaction.transactionType,
          zkAppPublicKey: dbSettings.appPublicKey,
          id: (transaction as any)._id,
          tx: transaction.tx,
        };
```

Reply:
You need to return id because your confirmTransaction function need to put id args

Reply:
done